Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Use a golang image as the base image
+FROM golang:1.19-alpine
+
+# Set necessary environment variables
+ENV CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the api server go module files into container, because the server depends on them
+COPY ./api /api
+
+# Copy the server go module files into container and download dependencies
+COPY server/go.mod server/go.sum ./
+RUN go mod download
+
+# Now, copy the source code into the container
+COPY . .
```

Comment:
I think we can be more specific here instead of copying everything, because you don't need the `kurtosis` folder and the `script` folder inside the image, right?

Reply:
even you don't need the Kurtosis package files inside the image

Reply:
ahhh yes, lemme see if I can only copy in the server code! great call out